npm/pkg/dataplane/dataplane_windows.go

```diff
@@ -65,6 +77,14 @@ func (dp *DataPlane) initializeDataPlane() error {
 	return nil
 }
 
+func marshalFilterMap(filtermap map[string]uint16) ([]byte, error) {
```

Comment:
There isn't really anything interesting happening in this function (since it's just JSON marshaling), so it's probably best to just inline this.

Reply:
made it inline again, @timraymond pls let me know if that looks better